### PR TITLE
Implemented working audio example with promise

### PR DESCRIPTION
**Pull request by TarikHuber**
A working PR based on @koenvb [PR](https://github.com/firebase/functions-samples/pull/254) for Primisified ffmpeg

### CODE REVIEW

File: ffmpeg-convert-audio/functions/index.js [hunk 1]
```diff
@@ -23,6 +23,19 @@ const fs = require('fs');
 const ffmpeg = require('fluent-ffmpeg');
 const ffmpeg_static = require('ffmpeg-static');
 
+function promisifyCommand(command) {
+  return new Promise(cb => {
```
Comment (nicolasgarnier) [inline ffmpeg-convert-audio/functions/index.js]: I think you could implement the Promise rejection like this:

```js
function promisifyCommand(command) {
  return new Promise((resolve, reject) => {
    command
      .on('end', () => {
        resolve(null);
      })
      .on('error', error => {
        reject(error);
      })
      .run();
   });
 };
```

**Comment by nicolasgarnier** [comment]
Can you also include the few lines added to the README please:

https://github.com/firebase/functions-samples/pull/254/files#diff-865d61ef09a041eeffe55825ca15d3ba

**Comment by nicolasgarnier** [comment]
Never mind since it's a small change I've updated your PR myself. Thanks for your help on this!

**Comment by TarikHuber** [comment]
Sorry @nicolasgarnier for the late response. Became father of twins few days ago. It's a little bit hard to find time for coding this days but next week I should be ON again 😄 thx for the fast fix.

**Pull request merged**